packages/bun-plugin-svelte/src/index.ts

```diff
@@ -25,36 +31,35 @@ function SveltePlugin(options: SvelteOptions = kEmptyObject as SvelteOptions): B
     setup(builder) {
       const { config = kEmptyObject as Partial<BuildConfig> } = builder;
       const baseCompileOptions = getBaseCompileOptions(options ?? (kEmptyObject as Partial<SvelteOptions>), config);
+      const baseModuleCompileOptions = getBaseModuleCompileOptions(
+        options ?? (kEmptyObject as Partial<SvelteOptions>),
+        config,
+      );
+
+      const ts = new Bun.Transpiler({
```

Comment:
this will do transforms that I worry will break either the CommonJS case or the otehr case.

Also, we should set target to the appropriate target - either browser or bun in this case.